README.md

```diff
@@ -31,7 +31,7 @@ After you're finished please remove all the comments and instructions!
   <img src="logo.png" alt="logo" width="140"  height="auto"/>
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Metrics webapp</b></h3>
 
 </div>
 
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-documentation:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online%20(add%20a%20link%20to%20the%20online%20version%20of%20your%20app%20to%20your%20README%20file).), Please provide a valid deployment link to your README file because the one you provided is broken.